hackathon/fixtures/hackathons.json

```diff
@@ -14,6 +14,7 @@
             "organiser":12,
```

Comment:
I think this as been resolved in #111 and now creates a merge conflict. Could you please resolve the merge conflict?

Reply:
Will do